source/git/gitsource.go

```diff
@@ -413,8 +488,14 @@ func git(ctx context.Context, dir string, args ...string) (*bytes.Buffer, error)
 		cmd.Dir = dir // some commands like submodule require this
 		buf := bytes.NewBuffer(nil)
 		errbuf := bytes.NewBuffer(nil)
+		cmd.Stdin = nil
 		cmd.Stdout = io.MultiWriter(stdout, buf)
 		cmd.Stderr = io.MultiWriter(stderr, errbuf)
+		cmd.Env = []string{
+			"PATH=" + os.Getenv("PATH"),
+			"GIT_TERMINAL_PROMPT=0",
+			//	"GIT_TRACE=1",
```

Comment:
nit: Remove commented env var.

Reply:
Left this in intentionally to show how to add more debug. Ideally, this would be configurable. Some places like this (eg. container logs and scheduler traces) are behind env vars but would prefer something more generic like `GODEBUG`